app/src/main/java/org/oppia/app/player/content/ContentCardAdapter.kt

```diff
@@ -0,0 +1,124 @@
+package org.oppia.app.player.content
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.databinding.library.baseAdapters.BR
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.content_card_items.view.*
+import org.oppia.app.R
+import org.oppia.app.databinding.ContentCardItemsBinding
+import org.oppia.app.databinding.LearnersCardItemBinding
+import org.oppia.data.backends.gae.model.GaeSubtitledHtml
+import org.oppia.util.data.UrlImageParser
+
+/** Adapter to bind the contents to the [RecyclerView].It handles rich-text content  */
```

Comment:
Update this comment to 
`/** Adapter to bind the contents to the [RecyclerView]. It handles rich-text content.  */`

Also, do check what differences are there in the two comments.

Reply:
updated comment added missing punctuation and space